### ISSUE TITLE
does this work run on windows

### ISSUE THREAD

**Issue by zaghdoud2019** [bgshih/aster#113]
I never working on ubuntu,its easy to run this code,any helps?

**Comment by old-school-kid** [comment]
Yeah it works in windows environment. You can clone the repo and then install the required packages using pip and run the code finally.
